### ISSUE TITLE
How to use

### ISSUE THREAD

**Issue by shu223** [jguertl/tensorflow-ios-models#1]
I tried to use the `flowers` model in [simple example](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/contrib/ios_examples/simple) which is provided by the official repository.

Please let me know how can I know what strings should be passed as `input_layer` or `output_layer` when using the flowers model?

```
    std::string input_layer = "input";
    std::string output_layer = "output";
    std::vector<tensorflow::Tensor> outputs;
    tensorflow::Status run_status = session->Run({{input_layer, image_tensor}},
                                                 {output_layer}, {}, &outputs);
```

For example, when using Inception model, those are `"input"` and `"output"`, but when I used this with flowers model, it caused an error.

> Running model failed: Not found: FeedInputs: unable to find feed output input

**Comment by jguertl** [comment]
Could you try it with the following settings?
![screen shot 2016-08-22 at 15 38 41](https://cloud.githubusercontent.com/assets/7988662/17856609/a11e713c-687e-11e6-8b8c-beb8061c526f.png)

**Comment by shu223** [comment]
Thanks, those settings seem to be ok! But, another error occurred...

> Running model failed: Invalid argument: computed output size would be negative
>      [[Node: pool_3 = AvgPool[T=DT_FLOAT, data_format="NHWC", ksize=[1, 8, 8, 1], padding="VALID", strides=[1, 1, 1, 1], _device="/job:localhost/replica:0/task:0/cpu:0"](mixed_10/join)]]

The input image is flowers.png which is included in this repository, and the size of `tensorflow::Tensor` is 224 \* 224, 3 channel, values are between 0.0 and 1.0. Do you have any idea what's wrong?

Or, could you push your working sample code into this repository?

**Comment by jguertl** [comment]
I will publish a blog post about running TensorFlow on iOS in the following days.

**Comment by jguertl** [comment]
New sizes:
![screen shot 2016-08-23 at 10 46 00](https://cloud.githubusercontent.com/assets/7988662/17885780/f82fcc32-691e-11e6-8c48-406c5b72fec8.png)

**Comment by shu223** [comment]
Thanks, I could go ahead, but there is another problem.

> Assertion failed: (this->rows()>0 && this->cols()>0 && "you are using an empty matrix"), function redux, file /tensorflow/contrib/makefile/downloads/eigen-eigen-b4fa9622b809/unsupported/Eigen/CXX11/../../../Eigen/src/Core/Redux.h, line 413.

I don't know it relates above assertion error, however the console message says:

> Op BatchNormWithGlobalNormalization is deprecated. It will cease to work in GraphDef version 9. Use tf.nn.batch_normalization().

I'm using TensorFlow 0.10.0 RC0. Is this ok?

**Comment by jguertl** [comment]
Are you using the model from my repository?

**Comment by shu223** [comment]
Yes, of course.
- model: tensorflow_flower_graph.pb
- labels: flowers_comp_graph_label_strings.txt
- input image: flowers.png

**Comment by shu223** [comment]
ok, i have an idea, i'll send a PR which includes my code. You can check whether something is wrong or not, and can even use it as the sample code of your model (if you want).

**Comment by shu223** [comment]
I recreated my project from the latest google source code, then it works fine now! Sorry, something in my codes seemed wrong.I'll send a PR with the example, so if you like that, please merge it :)  Thanks for your support!

**Comment by bibhu107** [comment]
@shu223  @jguertl 
Facing the same error.
```
Invalid argument: Computed output size would be negative: -32 [input_size: 84, effective_filter_size: 117, stride: 1]
Invalid argument: Computed output size would be negative: -32 [input_size: 85, effective_filter_size: 118, stride: 1]
Invalid argument:Computed output size would be negative: -32 [input_size: 86, effective_filter_size: 119, stride: 1]
Failed to run TensorFlow inference with inputs:[inputTensor, dropout_keep_prob], outputs:[output/softmax]
```
